mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -113,3 +113,45 @@ Description of the change and how users should adjust their configuration (if ne
 ### `deletion_policy` now defaults to `DELETE`
 
 Previously, `google_firebase_web_app` deletions default to `ABANDON`, which means to only stop tracking the WebApp in Terraform. The actual app is not deleted from the Firebase project. If you are relying on this behavior, set `deletion_policy` to `ABANDON` explicitly in the new version.
+
+## Resource: `google_firebase_project_location`
+
+### Deprecated
+
+The concept of a default location is no longer applicable. Consult the supported locations
```

Comment:
After looking over the docs for a while, I think this might be a better way to introduce the difference (similar to [this note from the 4.0 release](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_3_upgrade#ip_allocation_policy-will-catch-out-of-band-changes-use_ip_aliases-removed).)

```suggestion
In `4.X`, `google_firebase_project_location` would implicitly trigger creation of an App Engine application with a default Cloud Storage bucket, located in the specified `location_id`. With `5.0.0`, all resources should instead be set up explicitly.
```

Does that sound accurate? If so, I'd recommend also incorporating language from [this other note from 4.0 related to halting implicit resource creation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_3_upgrade#automatic-subnetwork-creation-for-vpc-native-clusters-removed). In particular, the guide should mention that the implicitly-created resources need to be imported.

Reply:
IC, let me specify the migration path.